### PR TITLE
[viewer] Support for URL based loading of scenes

### PR DESCRIPTION
**Pull request by apsdehal**
## Motivation and Context

If we are not on localhost, set the default scene to load from URL.

## How Has This Been Tested

Tested by deploying on aihabitat.org

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/esp/bindings_js/modules/defaults.js [hunk 1]
```diff
@@ -29,4 +29,7 @@ export const defaultEpisode = {
 
 export const defaultResolution = { height: 480, width: 640 };
 
-export const defaultScene = "skokloster-castle.glb";
+export const defaultScene =
+  window.location.href.indexOf("localhost") === -1
+    ? "https://habitat-demo.s3-us-west-2.amazonaws.com/data/scene_datasets/habitat-test-scenes/skokloster-castle.glb"
```
Comment (mathfac) [inline src/esp/bindings_js/modules/defaults.js]: can we make this constants? To make it easier to change default scene.

File: src/esp/bindings_js/bindings.css [hunk 2]
```diff
@@ -30,5 +30,5 @@
 }
 
 .hidden {
-  display: none;
+  display: none !important;
```
Comment (msbaines) [inline src/esp/bindings_js/bindings.css]: Why is !important needed?

File: src/esp/bindings_js/modules/defaults.js (hunk 1 above)
Comment (apsdehal) [reply src/esp/bindings_js/modules/defaults.js]: There are constants.

File: src/esp/bindings_js/bindings.css (hunk 2 above)
Comment (apsdehal) [reply src/esp/bindings_js/bindings.css]: Magnum somehow forces `style="display: block"` on the pre element.

File: src/esp/bindings_js/modules/defaults.js (hunk 1 above)
Comment (apsdehal) [reply src/esp/bindings_js/modules/defaults.js]: I meant that they can properly formatted, but defaults.js for us acts as constants.

**Pull request merged**